$NetBSD: patch-servo_ports_geckolib_cbindgen.toml,v 1.1 2024/09/18 07:27:42 wiz Exp $

Bug 1912663 - Fix some build issues with cbindgen 0.27. r=firefox-style-system-reviewers,zrhoffman

It updates serde and syn and they are more strict. In particular, syn 2
doesn't parse the rust 2015 syntax where try is not a keyword, and serde
rejects duplicate keys.

Differential Revision: https://phabricator.services.mozilla.com/D219025

--- servo/ports/geckolib/cbindgen.toml.orig	2024-08-26 14:25:35.000000000 +0000
+++ servo/ports/geckolib/cbindgen.toml
@@ -318,7 +318,6 @@ renaming_overrides_prefixing = true
 "Keyframe" = "Keyframe"
 "nsChangeHint" = "nsChangeHint"
 "ServoElementSnapshotTable" = "ServoElementSnapshotTable"
-"Keyframe" = "Keyframe"
 "ComputedKeyframeValues" = "ComputedKeyframeValues"
 "OriginFlags" = "OriginFlags"
 "ServoTraversalFlags" = "ServoTraversalFlags"
